ivy/functional/ivy/layers.py

```diff
@@ -1217,12 +1217,12 @@ def depthwise_conv2d(
 def conv3d(
     x: Union[ivy.Array, ivy.NativeArray],
```

Comment:
I see that in the docstring examples you used Container classes as inputs. If the function can accept them, then we need to add them here as well.